docs/products/postgresql/howto/postgresql-debezium-cdc.rst

```diff
@@ -0,0 +1,192 @@
+Handling PostgreSQL® node replacements when using Debezium for Change Data Capture
+=================================================================================
+
+As an Aiven customer you may be running a Aiven for Apache Kafka® Connect
```

Comment:
We can reference the devportal article about debezium doc https://docs.aiven.io/docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.html